trainings/admin-webinar/lesson-plan.md

```diff
@@ -177,7 +177,18 @@ The admin section of the Sourcegraph site offers a variety of subsections that c
 
 **Learning goals:** After completing this unit, the customer will be able to export user activity and understand how to view a list of users.
 
-### Viewing user activity
+### Viewing aggregate user activity
+
+To view aggregate user activity, go to the Users section in the Analytics section of the site admin panel. This shows you total active users, total registered users, and provisioned user licenses. The line graph at the top will show you user actions triggered over time or unique users using the app over time, toggled below the graph. You can also see DAU, WAU, and MAU at a glance, averaged out for the time frame you're viewing. Additionally, you'll see a bar graph showing how many days users used the application over the time period you specified—if you're seeing users mostly using Sourcegraph 1 or 2 days per time period, you may want to discuss how to reengage them with your CE.
+
+🔎 The trainer should show:
+
+* How to access the Users Analytics page
+* How to interpret the graph data
+
+❗️ The Usage Stats page has some of the same data; if asked, you can be honest with the user that the User Analytics page is a reimagining of the aggregate data in alignment with our overall value prop, but the Usage Stats page is still helpful to see individual activity.
```

Comment:
One other callout here - The usage-stats page is still the page which should be used when determining seat-license usage in context of billing. The legacy usage stats page will still be used for this until we move the new analytics out of 'experimental' stage.

Reply:
In general we don't encourage customers to use in-app data for billing, at all (we discuss the bill with them using data in Looker). (That said the data they need is actually on the License or Overview page; Usage Stats doesn't show the current registered user count, which is how we bill.)

See [Slack](https://sourcegraph.slack.com/archives/C03CSAER9LK/p1658768307987479) for why this is our general stance. :)

Reply:
@sourcegraph/customer-engineering Do you think this warrants its own callout for newer team members, or is our onboarding clear enough that we use ping data for this?